doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -38,6 +38,10 @@ nRF9160
     * LwM2M sensor objects now uses the actual sensors available to the Thingy:91. If the nRF9160 DK is used, it uses simulated sensors instead.
     * Added possibility to poll sensors and notify the server if the measured changes are large enough.
 
+  * :ref:`lib_lwm2m_client_utils` library:
+
+    * Added support for Firmware update object to use :ref:`lib_fota_download` library for downloading firmware images.
```

Comment:
Should it be stated that this is about HTTP.

Reply:
It is not limited to any protocol, at least not on this side.
Download is handled by [Download client](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/libraries/networking/download_client.html#protocols).
> The library supports HTTP, HTTPS (TLS 1.2), CoAP, and CoAPS (DTLS 1.2) over IPv4 and IPv6.

So I think it is enough that we point to the library that handles it. That library can be extended in the future.